unlock-app/src/components/interface/checkout/UnlockAccount/EnterEmail.tsx

```diff
@@ -60,18 +64,26 @@ export function EnterEmail({ unlockAccountService }: Props) {
             })}
           />
         </form>
-        <p className="ml-0.5 text-xs">
-          Use an Unlock account if you do not have your own crypto wallet.{' '}
-          <a
+        <p className="ml-0.5 text-sm grow">
+          This step enables you to log-in or create an{' '}
+          <Link
             href="https://docs.unlock-protocol.com/tools/sign-in-with-ethereum/unlock-accounts"
             target="_blank"
             rel="noopener noreferrer"
-            className="inline-flex items-center gap-1 text-gray-500 underline"
+            className="underline text-ui-main-500"
           >
-            <span>Learn more</span>
-            <ExternalLinkIcon />
-          </a>
+            Unlock account
+          </Link>
+          . If you already have a crypto wallet, please connect it.
         </p>
+        <section className="ml-0.5 text-sm mt-auto	">
```

Comment:
What do you think about de-emphaizing this using gray text? 🤔

Reply:
Also having a hierarchy here. Did you know could be bolder even if same size?

Reply:
Great ideas. Let me tweak that.